docs/DeveloperGuide.md

```diff
@@ -346,11 +346,11 @@ Step 4. If the format is correct, `RemarkCommandParser` will create a `RemarkCom
 
 Step 5. The `LogicManager` executes the `RemarkCommand`.
 
-Step 6. The `RemarkCommand` obtains a list of `Internship`s via the `getFilteredInternshipList()` method
-which is exposed to the `Model` interface as `Model#getFilteredInternshipList()`
+Step 6. The `RemarkCommand` obtains a list of `Internship`s via the `getSelectedInternshipList()` method
+which is exposed to the `Model` interface as `Model#getSelectedInternshipList()`
```

Comment:
Maybe you could include a step where if nothing is selected, an error will be thrown here for more detail!